### PR TITLE
Show version string in app

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is an attempt to show version string somewhere in the app.

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)? [🐱](https://jira.stsci.edu/browse/JDAT-3087)

### CODE REVIEW

File: jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue [hunk 1]
```diff
@@ -4,6 +4,10 @@
     :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#metadata-viewer'"
     :popout_button="popout_button">
 
+    <v-row no-gutters>
+      <span>Jdaviz {{ version }}</span>
```
[the remark this answers is not in the document]
Comment (kecnry) [reply jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue:8]: the plugin doesn't have direct access to traitlets defined at the app-level

[the remark this answers is not in the document]
Comment (pllim) [reply jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue:8]: Then how is it able to understand `vdocs` above?

[the remark this answers is not in the document]
Comment (kecnry) [reply jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue:8]: they need to be defined in `TemplateMixin` (available to the app and plugins) or `PluginTemplateMixin` (only available to plugins).  I don't see a standalone PR, it must have been in another PR and then we stripped that part out... so its probably easier to just rewrite.

**Comment by pllim** [comment]
Ah, so I was looking in the wrong module. Thanks, @kecnry !

Not sure if this is really what we want but at least something is showing. :shrug:

Screenshot 2023-02-09 095929

**Comment by pllim** [comment]
Better idea at #2049

**Pull request closed without merging**